src/headless_browser.ts

```diff
@@ -208,6 +213,9 @@ export class HeadlessBrowser {
       };
     };
     const actualUrl = res.root.documentURL;
+    if (actualUrl !== expectedUrl) { // Before we know the test will fail, close everything
+      await this.done();
```

Comment:
should we throw an error at all?

Reply:
How ddo you mean throw an error? Oh are you saying we should esentially turn this method into just `getUrl()`?, then the user would do:

```
const url = await Sinco.getUrl()
assertEquals(url, ...) // <-- The USER does the assertions
```

But atm, all that method does is just makes an assertions really